### PR TITLE
Prevent drag/dop and text selection on most elements.

### PR DESCRIPTION
**Pull request by hydra**
I've noticed, over the years, that it was possible to accidentally start a drag-and-drop operation when clicking various elements in the UI, something that shouldn't be possible and certainly isn't possible in native apps.

Similarly, when selecting text it was possible to accidentally select the text of UI elements, again something that shouldn't be possible unless specifically enabled.

This PR:
* disables text selection for everywhere except the CLI and the LOG.
* disables drag/drop for all elements.

### CODE REVIEW

File: src/css/main.css
Comment (McGiverGim) [inline src/css/main.css:33]: Join this with the one of line 25.

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/css/main.css
Comment (hydra) [reply src/css/main.css]: done

**Comment by hydra** [comment]
@McGiverGim is there a CSS selector that selects *ALL* children, not just direct descendants?  If so I could improve the selectors a bit.

**Review by McGiverGim — APPROVED** [review]

**Comment by McGiverGim** [comment]
> @McGiverGim is there a CSS selector that selects ALL children, not just direct descendants? If so I could improve the selectors a bit. e.g. `<strong>` and `<span>` elements within the LOG `<p>` elements, and `<br>` elements within the CLI <div>.

@hydra I don' t understand exactly... something like:

```css
tag1 tag2 {
...
}
```
will affect all the tag2 inside a tag1, but it does not need to be a direct descendant.

**Review by haslinghuis — DISMISSED** [review]
@Asizon can you confirm, as CSS is your thing :smile:

**Comment by hydra** [comment]
I need to fix the OSD drag/drop.  Don't merge this yet.  I forgot about that.

**Comment by hydra** [comment]
@McGiverGim ok, fixed with an additional commit.

**Review by McGiverGim — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Comment by haslinghuis** [comment]
@hydra can you please squash your commits?

**Comment by blckmn** [comment]
AUTOMERGE: (pass)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:26:17) -> pass
 - commit count less or equal to three (3 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> pass
 - assigned to an approver (haslinghuis) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:45:06) -> pass
 - commit count less or equal to three (3 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> pass
 - assigned to an approver (haslinghuis) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - Github identifies PR as mergeable -> pass
 - PR is not in draft -> pass
 - Assigned to a milestone -> pass
 - cooling off period lapsed (10/06/2021 00:48:45) -> pass
 - commit count less or equal to three (3 commits) -> pass
 - 'Do not merge' label NOT found -> pass
 - 'Needs coordination with betaflight' label NOT found -> pass
 - At least one 'RN:' label found -> pass
 - 'Tested' label found -> pass
 - assigned to an approver (haslinghuis) -> pass
 - approver count at least three (3 approvers) -> pass

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed (10/06/2021 00:52:14) -> PASS
 - commit count less or equal to three (3 commits) -> PASS
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> PASS
 - assigned to an approver (haslinghuis) -> PASS
 - approver count at least three (3 approvers) -> PASS

**Pull request merged**
